server/src-lib/Hasura/GraphQL/Execute/Query.hs

```diff
@@ -294,6 +295,35 @@ convertQuerySelSet env logger gqlContext userInfo manager reqHeaders directives
       AQAsync s -> AQPAsyncQuery <$>
         DS.traverseAnnSimpleSelect prepareWithPlan (resolveAsyncActionQuery userInfo s)
 
+-- | A pared-down version of 'convertQuerySelSet', for use in execution of
+-- special case of SQL function mutations (see 'MDBFunction').
```

Comment:
I feel a bit iffy about adding a function, `convertFunction`, which is only used in `Hasura.GraphQL.Execute.Mutation`, but defined in `Hasura.GraphQL.Execute.Query`. It suggests there should be a module called `Hasura.GraphQL.Execute.Common`. Or is there a different way to resolve this?

Speaking of `convertFunction`, is it possible to have remote joins as part of a mutation custom function?

Reply:
> I feel a bit iffy ...

Yeah, I think it would be better to leave that for a separate PR, post- https://github.com/hasura/graphql-engine/pull/5797/files#diff-563b07c689cd5e65fabcd9d3915cb42bb1cc065d507af4ded924dba3fca9408c 

> Speaking of convertFunction, is it possible to have remote joins as part of a mutation custom function?

Do you mean "does that work in the PR?" or "is it possible to make it work?" ? I don't know the answer to either, but can look into it. Maybe a future enhancement?